navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/instruction/LengthUnit.java

```diff
@@ -0,0 +1,97 @@
+package com.mapbox.services.android.navigation.v5.instruction;
+
+public class LengthUnit {
```

Comment:
This entire class can be removed in favor of `TurfHelpers.convertDistance()` added in this PR: https://github.com/mapbox/mapbox-java/pull/481